plugins/gatsby-theme-iterative-docs/src/components/Documentation/Markdown/index.tsx

```diff
@@ -50,11 +50,18 @@ const Details: React.FC<{ slugger: GithubSlugger }> = ({
     firstChild.props.children.length - 1
   ) as ReactNode[]
 
-  let slug = slugger.slug(triggerChildren.toString())
-  if (slug[0] === '️') {
-    slug = slug.slice(1)
-  }
-  const id = slug.startsWith('-') ? slug.slice(1) : slug
+  const title = (triggerChildren as any[]).reduce((acc, cur) => {
```

Comment:
Not sure why, but links with ⚙️ are now not working correctly: 

https://dvc-org-fix-details-obj-yq7a0c.herokuapp.com/doc/start/experiments#%EF%B8%8F-run-multiple-experiments-in-parallel

And it looks like both this branch and `dvc.org` details links don't work with  ℹ️:

https://dvc-org-fix-details-obj-yq7a0c.herokuapp.com/doc/start/experiments#%E2%84%B9%EF%B8%8F-more-information-about-metrics